_dprhtml/features/bottom-pane/main-pane.html

```diff
@@ -122,7 +122,42 @@
       </div>
     </div>
     <div id="main-bottom-pane-tab-panes-Cj" class="main-bottom-pane-tab-pane" data-bind="visible: isCjTabSelected">
-      Conjugation: Coming soon...
+      <form onsubmit="insertConj(); return false">
```

Comment:
in case you are interested with the css grid thing, which i think is an excellent thing to pick up for the future as well - let me know and we can have a quick chat on how to put this as a grid. plus i'll quickly transfer my understanding of grid/flexbox to you.

its super easy btw - just wondering why css took this loon to add this grid/flexbox.

Reply:
ok I have put this as a css grid